nixos/modules/services/monitoring/loki.nix

```diff
@@ -94,7 +94,8 @@ in {
     systemd.services.loki = {
       description = "Loki Service Daemon";
       wantedBy = [ "multi-user.target" ];
-      after = [ "network.target" ];
+      wants = [ "network-online.target" ];
+      after = [ "network.target" "network-online.target" ];
```

Comment:
Hmm, does this even need `network.target`? If you're after `network-online.target`, you're also after `network.target`. And I don't see why the latter is even needed here, it only means that the network stack is started (and doesn't say anything about actual ifs being up).